### PR TITLE
[v2.4] Routing: improvements, translations

### LINKED ISSUE #392: defaultní out filter
Dle commitu [448749a](https://github.com/nette/application/commit/448749a558eb4cf50c4bc6e3e31aa104597349df) bych řekl, že defaultním out filtrem není rawurldecode ale [param2path](https://github.com/nette/application/blob/master/src/Application/Routers/Route.php#L768-L771)

[Řádek kde je to asi špatně](https://github.com/nette/web-content/blob/doc-2.3/cs/routing.texy#L380)

### PR DESCRIPTION
**Pull request by f3l1x**
- resolve #392
- translate HTTPS caution

### CODE REVIEW

**Comment by TomasVotruba** [comment]
What needs to be done here?

**Comment by dg** [comment]
Fix this https://github.com/nette/docs/pull/498/files/c2936180357ec8407d719a23bddf2107dede1dfe#diff-341ad5e90dcdf6adb25df2328658e87c

**Comment by TomasVotruba** [comment]
@dg What exactly to fix tehre?

**Comment by dg** [comment]
I have feeling that authors of PR do not get information about my comments. Maybe I'm doing something wrong, maybe Github works bad, I do not know. I see this:

image

**Comment by TomasVotruba** [comment]
@dg Ah, I see. I'm glad you took the work and time and explain this to me. This is clearly UX fail of Github.
This happens to me as well, that's why I hate these "Reviews" feature.

This "pending" means pending for you, not for the author of PR. All your comments with "pending" **are actually in state of "concept"**. **You write them, but you have to publish them in the end with "Submit review".** After that, we all can see them.


*I feel you, because I've given some feedback on other PRs with this handing for weeks or even month. Blaming people, why they can't read :D  *

**Comment by dg** [comment]
Oh! Github really fucked it :-(

So there's a lot of my unsubmitted reviews… And I don't know how to find them.

**Comment by TomasVotruba** [comment]
@dg Well, they should be still ready-to-send in every PR. Highlighting slightly with that yellow color.

I can see you send it here now:

image

File: cs/routing.texy [hunk 1]
```diff
@@ -383,7 +383,7 @@ $route = new Route('<presenter>/<action>/<id>', [
 
 Kde `filterInFunc` a `filterOutFunc` jsou funkce či metody, jenž převádí mezi parametrem v URL a tvarem, který se předává do presenteru. Každá z nich zajišťuje převod opačným směrem.
 
-Implicitním in-filterem je funkce [rawurldecode | php:rawurldecode] a out-filterem funkce [rawurlencode | php:rawurlencode], která escapuje speciální znaky (například lomítko či mezeru) pro použití v URL.
+Implicitním in-filterem je funkce [rawurldecode | php:rawurldecode] a out-filterem funkce [param2path | api:Nette\Application\Routers\Route::_param2path], která escapuje speciální znaky (například lomítko či mezeru) pro použití v URL.
```
Comment (f3l1x) [inline cs/routing.texy]: @dg diky za comment btw ;-)

Comment (dg) [inline cs/routing.texy]: Prostě stačí *out-filterem je funkce, která escapuje speciální znaky*, prostě není potřeba uvádět její název a odkaz do API, když není součástí API.

**Comment by f3l1x** [comment]
@TomasVotruba @dg Fixed in latest version.

**Comment by dg** [comment]
Thanks

**Pull request merged**